chart/flux/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
-appVersion: "1.21.2"
-version: 1.6.2
+appVersion: "1.22.0"
```

Comment:
Shouldn't this be `1.22.1`, or is this the result of some other dance between release branches?

Reply:
That's right, the update in this branch is housekeeping from the last chart-bump. A new chart-bump branch is ready now and will merge this update once the release is published, in a separate PR.

That will be in #3456, which I just opened as a draft.